app/components/captcha_submit_button_component.rb

```diff
@@ -0,0 +1,43 @@
+class CaptchaSubmitButtonComponent < SpinnerButtonComponent
+  attr_reader :action, :tag_options
+
+  # @param [String] action https://developers.google.com/recaptcha/docs/v3#actions
+  def initialize(action:, **tag_options)
+    super(type: :button, big: true, wide: true)
+
+    @action = action
+    @tag_options = tag_options
+  end
+
+  alias_method :spinner_button_tag, :call
+
+  def call
+    button_tag + recaptcha_script_tag
+  end
+
+  def content
+    t('forms.buttons.submit.default')
+  end
+
+  private
+
+  RECAPTCHA_SCRIPT_SRC = 'https://www.google.com/recaptcha/api.js'.freeze
```

Comment:
would we ever want to stub a fake recaptcha in dev/sandbox/specs like we do for fake threatmetrix?

Reply:
> would we ever want to stub a fake recaptcha in dev/sandbox/specs like we do for fake threatmetrix?

That's an interesting idea! It'd definitely help with simulating outcomes, I could also imagine that for feature specs as well. Maybe something I'd want to address in a separate pull request though.

Reply:
Wrote a ticket: LG-8833